src/components/games/seabattle/game.ts

```diff
@@ -54,23 +54,33 @@ export const SeabattleGame = Game({
       const player = parseInt(ctx.playerID, 10);
       // Avoid another salvo if the last was a miss.
       if (G.salvos.length > 0 &&
-          G.salvos[G.salvos.length - 1].player ===  player &&
-          !G.salvos[G.salvos.length - 1].hit) {
+        G.salvos[G.salvos.length - 1].player === player &&
+        !G.salvos[G.salvos.length - 1].hit) {
         return { ...G };
       }
```

Comment:
This was dead code and was deleted in the last PR. Do a pull to make sure you are not using it.